upgrade/v2.5/script/crds.yaml

```diff
@@ -0,0 +1,69 @@
+kind: List
```

Comment:
I think it's better to have individual yaml documents separated with `---` here than to use a `kind: List`.

That seems to be the idiom most people use.